generators/app/index.js

```diff
@@ -80,6 +81,13 @@ class NgxGenerator extends Generator {
   }
 
   configuring() {
+    if (!this.props['i18n-langs']) {
+      this.props['i18n-langs'] = ['en-US'];
+    }
+    getLangs().forEach(v => {
+      this._prefixRules[v] = (props => props['i18n-langs'].indexOf(v) >= 0);
```

Comment:
This should be `.includes(...)` rather than the more obscure `.indexOf(...) >= 0`. See a few lines below.